content/docs/identity-providers/azure.mdx

```diff
@@ -1,156 +1,177 @@
 
-   ![Application dashboard](./img/azure/azure-application-dashboard.png)
+1. Go back to the **App registrations** page and select your app. Select the **Endpoints** tab. ![From the app registration page, select the Endpoints tab](./img/entra/entra-endpoints-tab.png)
 
-   Click on **Endpoints**
+   Copy the value under the **OpenID Connect metadata document** field:
 
-   ![Endpoint details](./img/azure/azure-endpoints.png)
+   ![The OpenID Connect (OIDC) connection endpoint metadata](./img/entra/entra-endpoint-metadata.png)
 
-The **OpenID Connect Metadata Document** value will form the basis for Pomerium's **Provider URL** setting.
+The **OpenID Connect metadata document** value will form the basis for Pomerium's **[Provider URL](/docs/reference/identity-provider-settings#identity-provider-url)** setting.
 
-For example if the **Azure OpenID Connect** url is:
+For example, if the **Entra ID OpenID Connect** URL is:
 
 ```bash
 https://login.microsoftonline.com/0303f438-3c5c-4190-9854-08d3eb31bd9f/v2.0/.well-known/openid-configuration`
 ```
 
-**Pomerium Identity Provider URL** would be
+Then the **Pomerium Identity Provider URL** would be:
 
 ```bash
 https://login.microsoftonline.com/0303f438-3c5c-4190-9854-08d3eb31bd9f/v2.0
 ```
 
 ## Pomerium Configuration
 
-Configure Pomerium with the identity provider settings retrieved in the previous steps.
+Configure Pomerium with the identity provider settings you saved in the previous steps:
 
 <Tabs>
-<TabItem value="config.yaml" label="config.yaml">
+<TabItem value="Config file keys" label="Config file keys">
 
 ```yaml
 idp_provider: 'azure'
-idp_provider_url: 'https://login.microsoftonline.com/{REPLACE-ME-SEE-ABOVE}/v2.0'
-idp_client_id: 'REPLACE-ME'
-idp_client_secret: 'REPLACE-ME'
+idp_provider_url: 'https://login.microsoftonline.com/{REPLACE-WITH-OIDC-METADATA-STRING}/v2.0'
+idp_client_id: 'REPLACE-WITH-CLIENT-ID'
+idp_client_secret: 'REPLACE-WITH-CLIENT-SECRET'
 ```
 
 </TabItem>
-<TabItem value="Environment Variables" label="Environment Variables">
+<TabItem value="Environment variables" label="Environment variables">
 
 ```bash
 IDP_PROVIDER="azure"
-IDP_PROVIDER_URL="https://login.microsoftonline.com/{REPLACE-ME-SEE-ABOVE}/v2.0"
-IDP_CLIENT_ID="REPLACE-ME"
-IDP_CLIENT_SECRET="REPLACE-ME"
+IDP_PROVIDER_URL="https://login.microsoftonline.com/{REPLACE-WITH-OIDC-METADATA-STRING}/v2.0"
+IDP_CLIENT_ID="REPLACE-WITH-CLIENT-ID"
+IDP_CLIENT_SECRET="REPLACE-WITH-CLIENT-SECRET"
 ```
 
 </TabItem>
 </Tabs>
 
+:::note
+
+As of this writing, Pomerium only supports **`azure`** as the valid **`idp_provider`** name to reference Microsoft Entra ID as an identity provider (not **`entra`**).
+
+This means you **do not** need to update **`azure`** in your configuration file. If you update the **`idp_provider`** key to **`entra`**, it will break your configuration.
```

Comment:
To me, the way this is phrased feels like we're assuming users have an existing configuration file with `azure` in it already. Given that this page is framed as a guide, I don't think this makes sense.

I'd suggest we stick to something simpler here:

```suggestion
Pomerium currently uses **`azure`** as the **`idp_provider`** name to refer to Microsoft Entra ID.
```